doc/mcuboot/index-ncs.rst

```diff
@@ -0,0 +1,68 @@
+.. _mcuboot_index_ncs:
+
+MCUboot
+#######
+
+.. contents::
+   :local:
+   :depth: 2
+
+MCUboot is a secure bootloader for microcontrollers.
+
+Overview
+********
+
+The goal of MCUboot is to define a common infrastructure for the bootloader, the system flash layout on microcontroller systems, and to provide a secure bootloader that enables easy software upgrade.
+
+MCUboot is operating-system and hardware independent, and relies on hardware porting layers from the operating system it works with.
```

Comment:
```suggestion
MCUboot is not dependent on any specific operating system and hardware and relies on hardware porting layers from the operating system it works with.
```
To avoid writing `operating-system- and hardware-independent`

Reply:
fixed